src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -88,7 +113,38 @@ private void submitTopicOperation(PathOperation op, OpType opType) {
     }
   }
 
-  private void submitTopicCreateOperation() {}
+  private void submitTopicCreateOperation(final PathOperation op) {
+
+    final String topicName = op.dstPath();
+    try {
+      val properties = new Properties();
+      try {
+        properties.load(new FileReader(new File("kafka.properties")));
+      } catch (IOException e) {
+        e.printStackTrace();
+      }
+      val adminClientCreateFunc = adminClientCreateFuncCache.computeIfAbsent(properties,
+              AdminClientCreateFunctionImpl::new);
+      val adminClient = adminClientCache.computeIfAbsent("kafka", adminClientCreateFunc);
+      val newTopic = new NewTopic(topicName, 1, (short) 1);
+      final CreateTopicsResult createTopicsResult = adminClient.createTopics(Collections.singleton(newTopic));
+      createTopicsResult.values().get(topicName).get();
```

Comment:
use a handler for the non-blocking execution

Reply:
fixed. used whenComplete()